project-init: markup of nav-bar and header

AksultanAitkozha opened the pull request.

AksultanAitkozha committed:
project-init: markup of nav-bar and header

bikalay commented on dist/index.html:
```diff
@@ -0,0 +1 @@
+<!doctype html><html lang="en"><head><meta charset="utf-8"/><title>Team Weekly Report</title><script defer="defer" src="main.js"></script></head><body class="h-100"><div class="h-100 d-flex"><div><aside class="aside"><img class="logo my-5 mx-4" alt="logo" height="45" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAKcAAAAtCAYAAADLPXtSAAAPLUlEQVR4Xu2ca2wc1RWAz5mZje0EEqr4sQ4goI4NXicBO1GrItGGVDbs ...
```
Please add dust folder to gitignore

AksultanAitkozha committed:
feat: added dist to .gitignore and updated header/navbar

Menu and header are components we can apply for all HTML pages, whilst the content is wrapped in a container and is unique for every page.  
Also, dist is in .gitignore now.
